Assets/__Scripts/Settings/Settings.cs

```diff
@@ -83,6 +83,10 @@ public class Settings {
     public bool Reset360DisplayOnCompleteTurn = true;
     public string Language = "en";
 
+    public string LastLoadedMap = "";
+    public string LastLoadedChar = "";
+    public string LastLoadedDiff = "";
```

Comment:
Is there any reason why we are saving them to the Config file?

Reply:
Matches up with MMA2's "Open Last" function (not actually added a button for that) but with the expectation that you're working on one map at a time you can get back to the part you were working on quickly even if you close the app. I could have written separate persistence but the Settings class is so convenient.